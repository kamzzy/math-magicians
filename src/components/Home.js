import React from 'react';

const Home = () => (
  <div className="homeDiv">
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      Morbi maximus et risus id tristique. Quisque aliquam ultricies
      commodo. Class aptent taciti sociosqu ad litora torquent per
      conubia nostra, per inceptos himenaeos. Vestibulum eget quam
      nec odio interdum dictum vel sed felis.
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      Morbi maximus et risus id tristique. Quisque aliquam ultricies
      commodo. Class aptent taciti sociosqu ad litora torquent per
      conubia nostra, per inceptos himenaeos. Vestibulum eget quam
      nec odio interdum dictum vel sed felis.
    </p>
  </div>
);

export default Home;
