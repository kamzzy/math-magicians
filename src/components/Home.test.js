import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from './Home';

describe('render Home page', () => {
  test('render p element content', () => {
    render(<Home />);
    const p = screen.findAllByText('Lorem ipsum', { exact: false });
    expect(p).toMatchSnapshot();
  });
  test('render h2 elemnt content', () => {
    const h2 = screen.findByText('Welcome');
    expect(h2).toMatchSnapshot();
  });
});
