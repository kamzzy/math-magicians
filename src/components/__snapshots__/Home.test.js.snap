// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home page render h2 elemnt content 1`] = `Promise {}`;

exports[`render Home page render p element content 1`] = `Promise {}`;
